nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -102,6 +102,17 @@ let
     proxy_set_header        X-Forwarded-Server $host;
   '';
 
+  cfgProxyCachePath = concatStringsSep "\n" (mapAttrsToList (name: proxyCachePath: ''
```

Comment:
following the convention, it should be called `proxyCachePathConfig` of this file

Reply:
This was not solved

Reply:
Fixed!